### DISCUSSION TITLE
windows病毒和威胁防护自动删除PCL快照版

### DISCUSSION THREAD

**Discussion by Gernica** [Meloong-Git/PCL#3156] [提问] [answered]
Windows病毒和威胁防护把PCL快照版视为威胁，并会自动删除，即使选择“允许在电脑上”，也会在几天后删除。

**Comment by allMagicNB** [comment, marked as answer]
**此外，由于 Windows 将 PCL 误报为病毒，如果在下载或更新时被拦截，可以在任务栏搜索 “病毒和威胁防护”，然后在管理设置的排除项中，把 PCL 所在的 整个文件夹 加入到排除项里。我在一周前就提交了误报反馈，但直到现在微软都还拖着没回复，加了数字签名也没用，实在万策尽了 Orz……**

龙猫在专栏中提供的解决方法
